src/components/artworks/img-gallery.css

```diff
@@ -49,20 +49,20 @@
 }
 
 .sliderWrap .btnClose {
-  top: 40px;
-  right: 40px;
+  top: 45px;
+  right: 45px;
 }
 
 .sliderWrap .btnPrev {
-  top: 95%;
+  top: 87%;
```

Comment:
I hope using % will work for mobile responsiveness...something to keep an eye on